src/hooks/useWrapCallback.ts

```diff
@@ -53,14 +58,17 @@ export default function useWrapCallback(
         inputError: sufficientBalance ? undefined : 'Insufficient ETH balance'
       }
     } else if (currencyEquals(WETH[chainId], inputCurrency) && outputCurrency === ETHER) {
+      const { ether, weth } = getChainCurrencySymbols()
       return {
         wrapType: WrapType.UNWRAP,
         execute:
           sufficientBalance && inputAmount
             ? async () => {
                 try {
                   const txReceipt = await wethContract.withdraw(`0x${inputAmount.raw.toString(16)}`)
-                  addTransaction(txReceipt, { summary: `Unwrap ${inputAmount.toSignificant(6)} WETH to ETH` })
+                  addTransaction(txReceipt, {
+                    summary: `Unwrap ${inputAmount.toSignificant(6)} ${weth} to ${ether}`
```

Comment:
also here, concerned that weth can be undefined

Reply:
same as above, otherwise what would be the best approach. I can make the check and don't allow to trade if this label is not defined. But I think this would be overcomplicating for the label, and is not that bad to assume that having a selected network we have a value there

happy to do the check if this is concerning